source/_integrations/uptimerobot.markdown

```diff
@@ -23,4 +24,6 @@ The `uptimerobot` integration provides entities to get the status for all of you
 
 To get your API key, go to [My Settings](https://uptimerobot.com/dashboard#mySettings) on the UptimeRobot website, at the bottom you will find your "Read-Only API Key".
 
+**NOTE**: If you want to be able to use the active monitor switch then you need to find your "Main API Key".
+
```

Comment:
```suggestion
```
Change the line above to "Main API Key" instead.

Reply:
Done.
Due to the changed logic in the integration, your request makes sense.